api.py

```diff
@@ -287,6 +287,13 @@ def version_root(req: Request):
     return "\n".join(['<ul>'] + lis + ['</ul>'])
 
 
+@v1.get("/collections", tags=["data"])
+@v1.head("/collections", include_in_schema=False)
+def get_collections(req:Request):
+    return [i for index in ES.indices.get('*')]
```

Comment:
For search purposes don't you think we should be getting indices prefixed with our ELASTICSEARCH_INDEX_PREFIX, a `*` may result in `dirty indices` or indices created by certain API actions